rbreslow/docker-garrysmodds
# Run SRCDS as steam user

## Overview

- Configure dependent images to run under `steam` (with [`USER`](https://docs.docker.com/engine/reference/#user) block)
- Refactor README
- Run [shfmt](https://github.com/mvdan/sh)

Connects https://github.com/rbreslow/docker-steamcmd/pull/3

## Testing Instructions

Build the image against the latest docker-steamcmd:

```bash
CI=1 ./scripts/cibuild
```

Start SRCDS and confirm the deprecation notice is gone:

```bash
$ docker run -ti \
    -p 27015:27015 \
    -p 27015:27015/udp \
    --rm quay.io/rbreslow/garrysmodds:slim
Auto detecting CPU
Using default binary: ./srcds_linux
Server will auto-restart if there is a crash.
Couldn't load shader dll: game_shader_generic_garrysmod_srv.soConVarRef mat_dxlevel doesn't point to an existing ConVar
Game_srv.so loaded for "Garry's Mod"
Setting breakpad minidump AppID = 4000
Initializing Steam libraries for Workshop..
...
```
